common/topology/src/lib.rs

```diff
@@ -119,11 +119,15 @@ pub struct NymTopology {
 }
 
 impl NymTopology {
-    pub fn new(mixes: BTreeMap<MixLayer, Vec<mix::Node>>, gateways: Vec<gateway::Node>) -> Self {
+    pub fn new(
+        mixes: BTreeMap<MixLayer, Vec<mix::Node>>,
+        gateways: Vec<gateway::Node>,
+        described_nodes: Vec<DescribedNymNode>,
```

Comment:
unless I'm missing something, this new field doesn't seem to be used anywhere? you just assign it and kinda leave it

Reply:
I'm using it here to get nodes' key: https://github.com/nymtech/nym/blob/1663f75b31626627db9895139024f8758afa2bdc/common/topology/src/lib.rs#L203
I need the description during the update phase, afterwards I only need a list of all nodes (including inactive and blacklisted one).
I was using https://github.com/nymtech/nym/blob/1663f75b31626627db9895139024f8758afa2bdc/common/topology/src/lib.rs#L142 to assign it before, but it wasn't set everywhere because of that and it was leading to problems.